tox.ini

```diff
@@ -3,3 +3,6 @@ envlist = py26, py27, py32, py33, py34
 
 [testenv]
 commands = python tests.py
+
+[flake8]
+exclude=docs/conf.py
```

Comment:
Hm.  I usually put `[flake8]` in setup.cfg.

Can you also add `doctests = yes` to the `[flake8]` section?